src/main/java/org/spongepowered/mod/SpongeGame.java

```diff
@@ -109,11 +111,14 @@ public ServiceManager getServiceManager() {
     }
 
     @Override
-    public Scheduler getScheduler() {
+    public SynchronousScheduler getSyncScheduler() {
         return SyncScheduler.getInstance();
     }
 
     @Override
+    public AsynchronousScheduler getAsyncScheduler() { return AsyncScheduler.getInstance(); }
```

Comment:
Nod I don't know who started it either but I do not like one line functions.   Thanks IDEA.